docs/_data/sidebar.yml

```diff
@@ -214,15 +214,6 @@
         en: Overview
         ja: 作成
       url: /aura/writing
-- title:
-    en: Salesforce Functions
-    ja: Salesforce の機能
-  baseUrl: /functions/
-  children:
-    - title:
-        en: Overview
-        ja: 作成
-      url: /functions/overview
```

Comment:
Do we not also want to delete: https://github.com/forcedotcom/salesforcedx-vscode/blob/08a132e08929c83fbd1b9876d4bd682f095e371c/docs/_articles/en/functions/overview.md#L2?